templates/Makefile

```diff
@@ -1,62 +1,68 @@
-SHELL := /usr/bin/env bash
+SHELL = /usr/bin/env bash
```

Comment:
This made me look up info on the various assignments: https://stackoverflow.com/a/448939
Good to know!